doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -159,6 +181,18 @@ Here are how the workflows look like for the typical planned operations:
 
 ### 8.1. HA set creation
 
+The HA bring-up workflow is described below -
+  - The DPU starts out with it's initial HA scope Role as Dead.
+  - First the SDN controller pushes all configurations including the HA set and the HA scope with role set to Active but AdminState set to Disabled.
+  - Then the SDN controller starts the HA state-machine on the DPU by updating the HA scope AdminState to Enabled.
+  - DPU HA state transitions to Connecting and attempts to connect to its pair specified in the HA set.
+  - If the connection attempt is unsuccessful it moves to the InitializingToStandalone statei and waits for trigger from SDN controller.
```

Comment:
will be better to explain this trigger and why this is needed.

Reply:
also typo: "statei" :)

Reply:
fixed. added separate section for activate-role trigger